pkg/resources/schema.go

```diff
@@ -53,9 +53,9 @@ var schemaSchema = map[string]*schema.Schema{
 	"data_retention_days": {
 		Type:         schema.TypeInt,
 		Optional:     true,
-		Default:      1,
+		Default:      -1,
 		Description:  "Specifies the number of days for which Time Travel actions (CLONE and UNDROP) can be performed on the schema, as well as specifying the default Time Travel retention time for all tables created in the schema.",
```

Comment:
If we stick with the default set to -1:
- please add this to the description (and regen the docs)
- please add this to the migration guide